### DISCUSSION TITLE
Code is resulting in error

### DISCUSSION THREAD

**Discussion by y00ni** [test-ysc/test_name#7] [Question]
`here is all of my code
to show as an example
of the thing i am talking about
where i cannot get it to work`

Do you have any fixes for this? Thanks

**Comment by Lyonsjc** [comment]
how unfortunate for you

**Comment by y00ni** [comment]
I am responding via email. Can you see this?

On Thu, Aug 5, 2021 at 1:17 PM Lyonsjc ***@***.***> wrote:

> how unfortunate for you
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/test-ysc/test_name/discussions/7#discussioncomment-1135815>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AP7MCEFAGRSZ7YBDRF2PKNLT3LBSJANCNFSM44B5RMIQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
*Yooni Choi*
*Associate Product Manager*
Ad Hoc LLC | ***@***.***

**Comment by y00ni** [comment]
@Lyonsjc plz answer
